### ISSUE TITLE
Training methods

### ISSUE THREAD

**Issue by fky07** [GuangxingHan/Meta-Faster-R-CNN#35]
I have a question. For VOC datasets, they are not randomly divided into three splits. Instead, I am renting a server online. I can rent three graphics cards of the same model and train three splits separately to obtain the results. Is this okay?

**Comment by GuangxingHan** [comment]
Yes, you can run the experiments on the three splits in parallel
